### ISSUE TITLE
make a tunning ambience for usage before beginning or starting sections

### ISSUE THREAD

**Issue by ttm** [ttm/music#4]

**Comment by ttm** [comment]
Folded into #1.

A tuning ambience is a tuning being sounded — the drone an orchestra makes tuning up, establishing the centre before a section starts. It needs something to know *what* to sound, and #1 is where that comes from: it now decodes the *Tintinnalogia* bell-tuning rule into a stated algorithm (it is a major scale from the tenor, verified against all six of the book’s worked examples) and carries this as its second half.

Both were waiting on the same missing home. `music.structures` has permutations, peals and symmetry and no tunings at all, despite the README having claimed otherwise for years — corrected in #89.

Closing here; the work is not lost, it is specified better.
